### ISSUE TITLE
fail to run 'FlightTable' in controlsfx-samples

### ISSUE THREAD

**Issue by jimliuxyz** [controlsfx/controlsfx#1188]
SDK:java11

it failed because the resoures 'impl/org/controlsfx/table/filter.png' is not accessible.

so I modified the module-info.java to make it open, it worked!, but not prefect, the filter stops working after the column switch.

![image](https://user-images.githubusercontent.com/32408633/65819195-45ec0b80-e24c-11e9-8893-343385474d42.png)

# detail
```
> Task :controlsfx-samples:FlightTable.main() FAILED
Exception in Application start method
java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at javafx.graphics/com.sun.javafx.application.LauncherImpl.launchApplicationWithArgs(LauncherImpl.java:464)
	at javafx.graphics/com.sun.javafx.application.LauncherImpl.launchApplication(LauncherImpl.java:363)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.base/sun.launcher.LauncherHelper$FXHelper.main(LauncherHelper.java:1051)
Caused by: java.lang.RuntimeException: Exception in Application start method
	at javafx.graphics/com.sun.javafx.application.LauncherImpl.launchApplication1(LauncherImpl.java:900)
	at javafx.graphics/com.sun.javafx.application.LauncherImpl.lambda$launchApplication$2(LauncherImpl.java:195)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.ExceptionInInitializerError
	at org.controlsfx.controls/org.controlsfx.control.table.ColumnFilter.attachContextMenu(ColumnFilter.java:336)
	at org.controlsfx.controls/org.controlsfx.control.table.ColumnFilter.<init>(ColumnFilter.java:129)
	at org.controlsfx.controls/org.controlsfx.control.table.TableFilter.lambda$new$1(TableFilter.java:92)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
	at java.base/java.util.stream.Streams$StreamBuilderImpl.forEachRemaining(Streams.java:411)
	at java.base/java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:658)
	at java.base/java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:274)
	at java.base/java.util.Iterator.forEachRemaining(Iterator.java:133)
	at java.base/java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578)
	at org.controlsfx.controls/org.controlsfx.control.table.TableFilter.lambda$new$4(TableFilter.java:92)
	at javafx.base/com.sun.javafx.collections.ListListenerHelper$Generic.fireValueChangedEvent(ListListenerHelper.java:329)
	at javafx.base/com.sun.javafx.collections.ListListenerHelper.fireValueChangedEvent(ListListenerHelper.java:73)
	at javafx.base/javafx.collections.ObservableListBase.fireChange(ObservableListBase.java:233)
	at javafx.base/javafx.collections.ListChangeBuilder.commit(ListChangeBuilder.java:482)
	at javafx.base/javafx.collections.ListChangeBuilder.endChange(ListChangeBuilder.java:541)
	at javafx.base/javafx.collections.ObservableListBase.endChange(ObservableListBase.java:205)
	at javafx.base/javafx.collections.ModifiableObservableListBase.add(ModifiableObservableListBase.java:155)
	at java.base/java.util.AbstractList.add(AbstractList.java:111)
	at org.controlsfx.samples/org.controlsfx.samples.tablefilter.FlightTable.start(FlightTable.java:72)
	at javafx.graphics/com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$9(LauncherImpl.java:846)
	at javafx.graphics/com.sun.javafx.application.PlatformImpl.lambda$runAndWait$12(PlatformImpl.java:455)
	at javafx.graphics/com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:428)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at javafx.graphics/com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:427)
	at javafx.graphics/com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:96)
Caused by: java.lang.IllegalArgumentException: Invalid URL: Invalid URL or resource not found
	at javafx.graphics/javafx.scene.image.Image.validateUrl(Image.java:1107)
	at javafx.graphics/javafx.scene.image.Image.<init>(Image.java:617)
	at org.controlsfx.controls/org.controlsfx.control.table.FilterPanel.<clinit>(FilterPanel.java:72)
	... 30 more
Caused by: java.lang.IllegalArgumentException: Invalid URL or resource not found
	at javafx.graphics/javafx.scene.image.Image.validateUrl(Image.java:1099)
	... 32 more
Exception running application org.controlsfx.samples.tablefilter.FlightTable

Caused by: java.lang.RuntimeException: Exception in Application start method

Caused by: java.lang.ExceptionInInitializerError

Caused by: java.lang.IllegalArgumentException: Invalid URL: Invalid URL or resource not found

Caused by: java.lang.IllegalArgumentException: Invalid URL or resource not found

Execution failed for task ':controlsfx-samples:FlightTable.main()'.
> Process 'command '/Library/Java/JavaVirtualMachines/jdk-11.0.4.jdk/Contents/Home/bin/java'' finished with non-zero exit value 1



```

**Comment by danielpeintner** [comment]
Related to https://github.com/controlsfx/controlsfx/commit/e45bf242063d333d393ea8b22295d622dceb0c93 ?

**Comment by koppor** [comment]
This is issue https://github.com/controlsfx/controlsfx/issues/1135, isn't it? The solution PR https://github.com/controlsfx/controlsfx/pull/1142 is available for branches `controlfsfx:9.0.0` and `jfx-13`, but not master.
